includes/azure-app-configuration-cleanup.md

```diff
@@ -18,7 +18,7 @@ If you do not want to continue using the resources created in this article, dele
 
 1. Sign in to the [Azure portal](https://portal.azure.com), and select **Resource groups**.
 1. In the **Filter by name** box, enter the name of your resource group. 
-1. In the result list, select the resource group, and either right-click the row or use the **ellipsis** (**...**) button to open the context menu.
+1. In the result list, left-click the resource group name to see the overview of resource group.
```

Comment:
Please use "select" rather than "left-click" for accessibility reasons.

The end of the line should either say "to see the overview of **the** resource group" -- or you could remove it entirely.

Reply:
Oh! @ktoliver I just saw your comment :) Yes, that rewording is better. Please make the edit. Thanks!